velox/dwio/common/Writer.h

```diff
@@ -54,6 +54,12 @@ class Writer {
    *  Data can no longer be written.
    */
   virtual void close() = 0;
+
+  /**
+   *  Aborts the writing by closing the writer.
+   *  Data can no longer be written.
```

Comment:
Can you explain in more detail about what makes this method different from `close()`?  Is the buffered data in writer no longer valid so we cannot flush it like we do in close?

Reply:
close() would finish encoding, flush out and write summary (meta) to disk. 
abort() only needs to stop write and drop everything. 

And for writer of different formats, they could behave differently so `bool flush` might not be enough, which is why we need an extra function